contrib/mesos/pkg/scheduler/meta/annotations.go

```diff
@@ -25,7 +25,6 @@ const (
 	TaskIdKey                = "k8s.mesosphere.io/taskId"
 	SlaveIdKey               = "k8s.mesosphere.io/slaveId"
 	OfferIdKey               = "k8s.mesosphere.io/offerId"
-	ExecutorIdKey            = "k8s.mesosphere.io/executorId"
```

Comment:
Concerned that if we get rid of this we won't be able to send framework messages to an executor if the only context we have is the task. Doesn't look like we've needed this, yet.